### ISSUE TITLE
When re-running a WB analysis, not all images are removed

### ISSUE THREAD

**Issue by nicholst** [NISOx-BDI/SwE-toolbox#27]
When SwE detects that an Analysis directory has results, it issues a warning to the user and seeks confirmation before deleting existing files.

I just tested this, and found many images were not removed, in fact, because we haven't updated the filenames in the list of patterns to delete. (See https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_cp_WB.m#L66).

I can't certify this as a comprehensive list, but here are the particular images I found that were not removed:
```
swe_clustere_Fstat_lpFWE-WB_c01.nii
swe_vox_Fstat_c01.nii
swe_vox_Fstat_lp-WB_c01.nii
swe_vox_Fstat_lpFDR-WB_c01.nii
swe_vox_Fstat_lpFWE-WB_c01.nii
swe_vox_Fstat_lp_c01.nii
swe_vox_fit_y????.nii
swe_vox_mask.nii
swe_vox_resid_y????.nii
swe_vox_xFstat-WB_c01.nii
swe_vox_xFstat_c01.nii
```

Now, some of these are usually deleted by the end of a run (`swe_vox_fit_y????.nii` and `swe_vox_resid_y????.nii`) but, as can happen, this was an aborted analysis and so they weren't deleted.

The `files` variable needs to be updated, removing filenames that are no longer created and removing ones that are.

**Comment by TomMaullin** [comment]
Hi @nicholst ,

Ah okay - apologies - I thought this was the desired behaviour. At the end of an aborted analysis and at the end of a fully run analysis, which files should remain?

**Comment by nicholst** [comment]
Hi @TomMaullin,

I do *not* see the need to have some 'on interrupt' or 'on crash' system to clean up aborted analyses... that's more than anyone can expect.

*But*, it is true that at the start of an analysis, if existing results are found and the user confirms it's OK to continue, *all* the output of the previous analysis should be removed.  See, e.g. https://github.com/spm/spm12/blob/master/spm_spm.m#L373-L383 in spm_spm.m, analogous to our https://github.com/NISOx-BDI/SwE-toolbox/blob/master/swe_cp.m#L83-L92 

But this list of files to delete (at the start of a new run) must be comprehensive... e.g. you might be overwriting a WB analysis with a parametric run.   Right now the linst on swe_cp.m L83 is not comprehensive and in particular doesn't reflect our updated naming conventions.

Does this make sense?

**Comment by TomMaullin** [comment]
This has now been addressed in #35
